### DISCUSSION TITLE
Quick question regarding CoreferenceResolver

### DISCUSSION THREAD

**Discussion by staedi** [explosion/spaCy#13365] [Help: Other Questions] [answered]
Hello,
Before begin, thanks for the great (and my favorite NLP) product.

I found `CoreferenceResolver` module particularly useful while trying out with `spacy-experimental`.
Although it's currently available only in selected languages and within a `trf` model, it is very impressive that it achieves sufficiently what I wanted.

As `CoreferenceResolver` has been under the experimental stage, I wonder if we could expect it to be shipped in the normal release sometime in the future (not necessarily soon).

Thanks again.

**Comment by svlandeg** [comment, marked as answer]
Hi!

Thanks for the feedback - it's good to hear that the experimental `coref` model works well for your use-case. We do want to include the `coref` model into spaCy's core code base at some point (and into a "normal" pretrained pipeline), but considering other priorities it probably won't be in the immediate future...
